api/resources_portal/views/grant_user.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework import viewsets
+from rest_framework.exceptions import MethodNotAllowed
+from rest_framework.permissions import BasePermission, IsAuthenticated
+from rest_framework.response import Response
+
+from rest_framework_extensions.mixins import NestedViewSetMixin
+
+from resources_portal.models import Grant, GrantUserAssociation, User
+from resources_portal.views.relation_serializers import UserRelationSerializer
+
+
+class OwnsGrant(BasePermission):
```

Comment:
Can we make this OwnsGrantOrIsAdmin? Admins should be able to do everything the owner can.

Reply:
Yep! just done, also added a test for it.